AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx

```diff
@@ -109,9 +109,20 @@ export default class LocationReportSubmit extends React.Component {
             sendExternal: this.state.sendExternal
         };
 
-        apiPost('reports/locationstatuses', body)
-            .then(() => this.addMessage('Report submitted', 'info'))
-            .catch(() => this.addMessage('Error submitting report, please try again later', 'danger'));
+
+
+        apiPost('reports/locationstatuses', body);
+        if(this.state.status>100){
+            apiPost('status/reports',body)
+                .then(() => this.addMessage('Report submitted', 'info'))
+                .catch(() => this.addMessage('Error with status - make sure its less than or equal to 100', 'danger'));
+        }
+        else{
+            apiPost('reports/locationstatuses',body)
+                .then(()=> this.addMessage('Report submitted','info'))
+                .catch(()=> this.addMessage('Error submitting report, please try again later','danger'));
+        }
+
```

Comment:
Could you double-check the white-spacing on this? Spacing might seem nit-picky but it makes it easier to read the code, and that's highly important!

Reply:
I added the backend validation and fixed the white spacing.